waypoint_navigator_node:
  ros__parameters:
    start_id: 0
    loop_enable: true
    loop_count: 2